performance/src/gatling/simulations/org/pmiops/databrowser/Pages.scala

```diff
@@ -28,7 +28,7 @@ object Pages {
     val domainTotals: HttpRequestBuilder = http("domain-totals").get("/v1/databrowser/domain-totals")
     def domainSearch (searchTerm: String): HttpRequestBuilder = {
       def encodedTerm: String = URLEncoder.encode(searchTerm, StandardCharsets.UTF_8.toString).toLowerCase
-      http("domain-totals").get("/v1/databrowser/domain-totals?searchWord=" + encodedTerm)
+      http("domain-search").get("/v1/databrowser/domain-search?searchWord=" + encodedTerm)
```

Comment:
@rushtong on further review I think this was just a typo as domain-totals doesn't take any parameters. Looks like the intent was to cover these domain keyword searches